docs/docs/quickstart.rst

```diff
@@ -70,15 +71,39 @@ To export comments from your previous WordPress installation, go to *Tools*,
 export your data. It has been reported that WordPress may generate broken XML.
 Try to repair the file using ``xmllint`` before you continue with the import.
 
-Now import the XML dump:
+For any other comment system you can use the generic JSON format. It's up to you
```

Comment:
Can you perhaps move this to a separate file and link to it from the quickstart?

Reply:
I could create a new page about importing json files in docs/extra and add a link from the quickstart.

What do you think?

Reply:
That seems like a good idea